vendor-bin/eloquent/composer.json

```diff
@@ -5,8 +5,8 @@
         ]
     },
     "require-dev": {
-        "illuminate/database": "~5.5.0",
-        "illuminate/filesystem": "~5.5.0",
+        "illuminate/database": "~5.5.0 || ~6.20.0 || ~7.29.0",
+        "illuminate/filesystem": "~5.5.0 || ~6.20.0 || ~7.29.0",
```

Comment:
we can drop v5 & v6 here to make it easier

Reply:
Can we ? The v7 is only working with Symfony 5 to me https://github.com/illuminate/filesystem/blob/7.x/composer.json

Reply:
Done !